<launch>
  <node pkg="reactive_assistance" type="reactive_assistance_node" name="reactive_assistance_node" required="true" output="screen">
    <!-- Length and width of rectangular mobile base -->
    <param name="footprint_length" value="0.50"/>
    <param name="footprint_width" value="0.50"/>

    <!-- Robot's kinematics: 0.5 and 1 work fine -->
    <param name="max_lin_vel" value="1.5"/>
    <param name="max_ang_vel" value="1.5"/>

    <!-- Forward simulation time of trajectory for shared control -->
    <param name="sim_time" value="1.0"/>

    <!-- Parameter used in the admissible gap method -->
    <!-- <param name="dvel_safe" value="0.9"/> -->
    <param name="dvel_safe" value="0.4"/>

    <!-- Output command velocities can also be configured -->
    <param name="cmd_sub_topic" value="main_js_cmd_vel"/>
    <param name="goal_sub_topic" value="move_base_simple/goal"/> 

    <remap from="base_scan" to="scan"/>
    <remap from="auto_vel" to="cmd_vel"/>
    <remap from="odom" to="ground_truth/state"/>
  </node>  

    <!-- <node pkg="tf" type="static_transform_publisher" name="odom_2_world_tf" args="0 0 0 0 0 0 1 odom  world 60"/> -->

</launch>
